portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,15 @@
 /**
  * Adds a random greeting to the page.
  */
+document.getElementsByTagName("h1")[0].style.fontSize = "6vw";
```

Comment:
This seems already covered by CSS. Is there a reason you're setting it via JavaScript as well?

Reply:
saw it in an example online but realized i could just do it on the css itself